### ISSUE TITLE
Bleeding hollow ruler gets a claim on a Stranglethorn title

### ISSUE THREAD

**Issue by ElMariuso** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2#266]
- The Bleeding Hollow would need to have a claim on a Stranglethorn title in order to declare war on it.

- _During the First War against Stormwind, Kilrogg and the Bleeding Hollow led a few raiding parties into the western jungles of Stranglethorn Vale, as the region reminded them of their ancestral home and they had aims of claiming it for themselves. However, the Gurubashi jungle trolls who dwelled in the jungles united to fight back against this new threat and began a guerrilla campaign against the orcs. The fighting between the two groups was unspeakably vicious. While the Bleeding Hollow seemed to outnumber the Gurubashi in every skirmish, they were still at a severe disadvantage against the trolls' knowledge of the region. The Gurubashi were willing to give up vast swathes of jungle to draw the orcs deeper into their territory, only to then launch bloody ambushes from all directions, resulting in horrific losses for the Bleeding Hollow. When Blackhand learned how many orcs were dying in Stranglethorn for no apparent gain, he ordered the Bleeding Hollow to retreat and join the fight against the humans. The Horde would deal with the trolls later._

https://wowpedia.fandom.com/wiki/Bleeding_Hollow_clan#The_First_War

**Comment by zumbak04** [comment]
I would rephrase it a bit to talk in CK3 game terms. Instead of claim, I would say CB.
Also, #123 is similar to this issue.

I would suggest:
- [ ] Scripting the Bleeding Hollow AI to invade Stranglethorn as long as the Horde invades Azeroth and Khaz Modan.
- [ ] Making sure player Bleeding Hollow have a CB during the Horde Invasion
